python/delta-kernel-rust-sharing-wrapper/.gitignore

```diff
@@ -0,0 +1 @@
+Cargo.lock
```

Comment:
What's this for?

Reply:
When you run `cargo` commands or `maturin develop`, a `Cargo.lock` file may be generated. Adding it to the gitignore makes sure you don't accidentally add it to git.

Reply:
for further context, generally for libraries (like this) you don't want to commit whereas binaries you normally do